### DISCUSSION TITLE
Does winget support Powershell 5??

### DISCUSSION THREAD

**Discussion by Romanitho** [Romanitho/Winget-AutoUpdate#676] [Q&A]
![image](https://github.com/user-attachments/assets/f2c7e017-4190-42f3-9d74-0c6144ec838e)

**Comment by Romanitho** [comment]
It doesn't seem to work in system context...

**Reply by AndrewDemski-ad-gmail-com** [reply]
That Microsoft.WinGet.Client module uses "& winget" syntax in its code.

Maybe it is unable to find winget executable and that is why it fails?

**Reply by Romanitho** [reply]
probably yes.
